pages/user/index.html

```diff
@@ -256,7 +256,7 @@ <h2 class="fs-2 fw-bold" id="descriptionHeader">About Me</h2>
                       type="button"
                       class="imgSize img-fluid d-block mx-auto rounded-circle"
                       src="https://images.rawpixel.com/image_1300/czNmcy1wcml2YXRlL3Jhd3BpeGVsX2ltYWdlcy93ZWJzaXRlX2NvbnRlbnQvbHIvay01Ni1kc2MyOTY1NDU2NjQ1MzQ1NjM5LmpwZw.jpg"
-                      alt=""
+                      alt="Here is your company logo for you company profile"
```

Comment:
This might be too specific? it's not neccessarily true that they are using a company logo?

Reply:
This is a company form so maybe its best to set it as "Company Avatar"?

Reply:
yeah that sounds good!